cli/cmd/sync_v3.go

```diff
@@ -375,21 +362,41 @@ func syncConnectionV3(ctx context.Context, source v3source, destinations []v3des
 			SourceErrors:        totals.Errors,
 			SourceWarnings:      totals.Warnings,
 			SyncID:              uid,
+			SyncTime:            syncTime,
```

Comment:
Noticing this now but it seems there's a bug with `SourceErrors` and `SourceWarnings`. We add `m.Warnings` and `m.Errors` (the destination metrics) a bit before to `totals.Warnings` and `totals.Errors`, so we each loop accumulates all the warnings and errors

Reply:
Fixed in separate pr: https://github.com/cloudquery/cloudquery/pull/17914